src/OpenTelemetry.Contrib.Exporter.Stackdriver/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased
+
+* Renamed project to `OpenTelemetry.Exporter.Stackdriver`

```

Comment:
Lets be a bit more specific here:
The nuget package will named `OpenTelemetry.Exporter.Stackdrive" going forward. Add link-to-new one (It'll be a deadlink now, but once we publish it'll be fixed). The old versions can still be found in nuget.org/link-to-old-package.